ios/HybridViewModelBooleanProperty.swift

```diff
@@ -28,20 +28,10 @@ class HybridViewModelBooleanProperty: HybridViewModelBooleanPropertySpec {
   }
   
   func addListener(onChanged: @escaping (Bool) -> Void) throws {
-    let id = property.addListener({ value in
-      onChanged(value)
-    })
-    listenerIds.append(id)
-  }
-  
-  func removeListeners() throws {
-    for id in listenerIds {
-      property.removeListener(id)
+    helper.trackListener { property in
```

Comment:
Nice! 

Btw can't we make a helper.addListener?! that does call trackListener and property.addListener?!